### DISCUSSION TITLE
TeamPassConnect

### DISCUSSION THREAD

**Discussion by bambi1013** [nilsteampassnet/TeamPass#3723] [Q&A]
Bonjour,
Je découvre l'application TeamPass qui semble très complète et répondre à beaucoup d'attentes.

Afin de l'utiliser avec un navigateur, j'ai consulté cette page : https://teampass.net/teampassconnect.html

Après installation de TeamPassConnect sous Firefox 113, l'icône de l'outil n'apparait pas.
TeamPassConnect  est-il toujours d'actualité ?
Existe-t-il une autre extension pour les navigateurs ?

Cordialement
Eric

**Comment by nilsteampassnet** [comment]
Hello Éric,

TeampassConnect is not maintained anymore and is not updated for Teampass v3.
Focus is set on Teampass itself.
Once done, I will think about such browser extension.

**Comment by bambi1013** [comment]
Bonjour,
Merci pour votre réponse et pour le travail réalisé.
Cordialement
